CONTRIBUTING.md

```diff
@@ -36,49 +36,41 @@ the latest information.
 
 #### 1. Set up your Go environment
 
-This project is written in Go. To be able to contribute you will need a working Go installation. You can check the 
-[official installation guide](https://golang.org/doc/install). Make sure your `GOPATH` and `GOBIN` 
-[environment variables](https://github.com/golang/go/wiki/SettingGOPATH) are set correctly.
+This project is written in Go. To be able to contribute you will need:
 
-Here are some quick start steps you can follow:
+1. A working Go installation of Go >= 1.12. You can check the
+[official installation guide](https://golang.org/doc/install).
 
-Select a folder path for your go projects, for example `~/eksctl`
-```bash
-mkdir ~/eksctl
-cd ~/eksctl
-export GOPATH="$(pwd)"
-export GOBIN="${GOPATH}/bin"
-```
-
-> NOTE: Windows users should install Docker for Windows and run `make eksctl-image` to build their code.
+2. Make sure that `$(go env GOPATH)/bin` is in your shell's `PATH`. You can do so by
+   running `export PATH="$(go env GOPATH)/bin:$PATH"`
 
-> TODO: Improve Windows instructions, ensure `go build` works.
 
 #### 2. Fork and clone the repo
 
-Clone the repo using go:
+Make a fork of this repository and clone it by running:
 
 ```bash
-go get -d github.com/weaveworks/eksctl/...
+git clone git@github.com:<yourusername>/eksctl.git
 ```
 
-Make a fork of this repository and add it as a remote:
+It is discouraged to clone under `go env GOPATH`. Otherwise you will need to set 
```

Comment:
Sounds a bit weird and unclear... how about this?
```suggestion
It is not recommended to clone under `GOPATH`, otherwise you will need to set
```

Reply:
It's intentional, with `1.12` you don't need do `GOPATH` env variable anymore

Reply:
Although, we are losing clarify and friendliness for precision. I will change it.